src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -2715,7 +2717,7 @@ bool primitive_inst::is_valid_fusion() const {
                                                          cldnn::format::dimension(data_layout.format),
                                                          false);
 
-            if (gemm_dims[0] != data_dims[0])
+            if (gemm_dims[0] != data_dims[0] && outer_dep_pshape_count != 1)
```

Comment:
Do we need to check it?

Reply:
Yes, if the number of elements is not one, we cannot broadcast it implicitly.